packages/bodiless-demo/src/components/Video/AutoPlayback.tsx

```diff
@@ -0,0 +1,44 @@
+// from https://esausilva.com/2021/06/14/react-hook-to-play-video-using-intersection-observer/
+
+import { useRef, useEffect } from 'react';
+
+// @ts-ignore:  TS7006: Parameter 'entries' implicitly has an 'any' type.
+const useVideoAutoPlayback = options => {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const videoRef = useRef<HTMLVideoElement>(null);
+
+  // catching the unmounted error
+  if (!containerRef || !videoRef) return [null, null];
```

Comment:
Issue:  in edit interface -- if I hit reload or swapped between edit/preview, it sometimes hit unmounted error

Added this check if I both valid refs and then passed back null one of them wasn't valid.